app/controllers/DigitalSubscription.scala

```diff
@@ -150,6 +150,7 @@ class DigitalSubscription(
       lastName = request.lastName,
       country = request.country,
```

Comment:
Once `support-workers` gets country and state from billing address, I will remove country and state from `CreateSupportWorkersRequest`